projects/_posts/2024-06-22-ASD-Diganosis.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: project
+title: "ASD-Diagnosis: Developing Novel Deep Learning Models for ASD Diagnosis using Complex MRI Data"
+contributors: [Prof-S, falmuqhim]
+handle: ASD-Diagnosis
+status: collection
+type: dataset
+
+# Optional
+website:
+# grant: NSF TT-2213951
+# grant_url: https://www.nsf.gov/awardsearch/showAward?AWD_ID=2213951&HistoricalAwards=false
+image: /assets/images/projects/2024-06-22-asd-diagnosis.png
+# tagline: "Reference EEG Benchmark for Prediction of Epileptic Seizures"
+tags: [asd, fmri]
+
+# Data and code
+# github: https://github.com/pcdslab/MLSPred-Bench
```

Comment:
Give your ASD-SAENET GitHub link. Multiple GitHub links can be given at a later stage.